packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -86,8 +86,8 @@ export class TabBarRenderer extends TabBar.Renderer {
         const dataset = this.createTabDataset(data);
         return h.li(
             {
-                key, className, title: title.caption, style, dataset,
-                oncontextmenu: event => this.handleContextMenuEvent(event, title)
+                key, className, id: title.owner.id, title: title.caption, style, dataset,
```

Comment:
that's a bad idea, we need to add some prefix, otherwise there are more than one node with widget id